src/baseball_stats_python/statcast/minor_statcast_search.py

```diff
@@ -126,6 +130,9 @@ def minor_statcast_pitcher_search(
     if not pitchers_lookup:
         raise ValueError('pitchers_lookup is required')
 
+    if start_dt or end_dt:
```

Comment:
Only need one validation since `statcast_batter_search` and `statcast_pitcher_search` also call `statcast_search`